_posts/2023-04-09-haberdashers-win-sr2023.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Haberdashers' School wins Student Robotics Competition 2023!
+image: /images/content/blog/sr2022/sr2023-photo.JPG
+image_alt: All the teams from SR2023
+description: SR2023 came to an exciting conclusion with a victory for Haberdashers' School
+---
+
+{% include figure.html src="/images/content/blog/sr2023/sr2023-photo.JPG"
+           caption="All the teams from SR2023" %}
+
+That’s a wrap on Student Robotics 2023! [Haberdashers’ School](https://www.habsboys.org.uk/) came 1st amongst all of our competitors, after coming up against [Abingdon School](https://www.abingdon.org.uk/), [Gymnasium Markt Indersdorf](https://www.gym-indersdorf.de/), and [Harris Westminster Sixth Form](https://www.harriswestminstersixthform.org.uk/) in the finals. Our teams worked incredibly hard on their robots this year, and it showed in their performance in each round.
+
+## The game: Greed
+
+Greed required our team’s robots to capture tokens from their opponents and place them in their scoring zone. With Bronze, Silver, and Gold tokens, each worth a different number of game points, teams had to think carefully about their optimum strategy.
+
+To help teams locate the tokens there were 2D barcode style markers attached that our computer vision library could detect. However each of the markers identified themselves as exactly the same, so teams had to use other sensors to detect which type of token is which. Bronze and Silver tokens were the same size but the Silver tokens weighed 300g more, and Gold tokens were larger than both Bronze and Silver. Teams were scored at the end of the match for collecting the most points.
+
+{% include figure.html class="right" src="/images/content/blog/sr2023/arena.jpg"
+           caption="The 'Greed' arena" %}
+
+## Virtual Competition
```

Comment:
Maybe include a screenshot from the virtual competition to give a feel for that? (just a thought).